src/matching.py

```diff
@@ -388,13 +388,28 @@ class RuleStrategy:
     and they return None when there is no match
     """
 
+    @staticmethod
+    def match_verbatim(plans: list, verbatim_id: str, **kwargs):
+        """
+        Match exactly to a verbatim message's ID.
+        """
+        for plan in plans:
+            if plan["id"] == verbatim_id:
+                return {
+                    "match": verbatim_id,
+                    "confidence": 100,
+                    "plan": plan,
+                    "verbatim": True,
+                }
+
     @staticmethod
     def match_display_title(plans: list, post_text: str, **kwargs):
         """
         Exact display title matches. Include some preprocessing just to allow punctuation to be imperfect,
         or the user to include a stop word for some reason
         """
         preprocessed_post = Preprocess.preprocess_gensim_v1(post_text)
+        plans = filter(lambda p: p.get("display_title"), plans)
```

Comment:
I assume this is only needed because plans and verbatims have been combined?

Reply:
That's right.